tenant-cd-api/src/main/java/ai/vespa/hosted/cd/Deployment.java

```diff
@@ -11,4 +13,13 @@ public interface Deployment {
     /** Returns an Endpoint in the cluster with the given id. */
     Endpoint endpoint(String id);
 
+    /** The Vespa runtime version of the deployment, e.g., 8.16.32. */
+    String platformVersion();
+
+    /** The build number assigned to the application revision of the deployment, e.g., 496. */
+    long applicationVersion();
```

Comment:
Did we settle on a name for this? Build? Revision? Now version?

Reply:
Hmm yes ... Not sure why I felt this name was more appropriate here. I found `platform` too terse, and `applicationVersion` fits better with `platformVersion`. Not all three terms are used >_<